samples/sensor/icm42670S/src/main.c

```diff
@@ -214,6 +236,78 @@ int main(void)
  #ifdef CONFIG_ICM42670S_APEX_SMD
 			printf("[%s]: SMD\n", now_str());
  #endif
+#elif defined(CONFIG_ICM42670S_AML)
+ #ifdef CONFIG_ICM42670S_AML_POINTING
+			struct sensor_value pointing_data;
+			sensor_channel_get(dev, SENSOR_CHAN_AML,
+							&pointing_data);
+			
+			printf("[%s]: AML Pointing Delta=[%d, %d]\n",
+				   now_str(),
+				   pointing_data.val1, pointing_data.val2);
+ #endif
+ #ifdef CONFIG_ICM42670S_AML_GESTURES
+ 			struct sensor_value aml_data[2];
+			static uint8_t previous_position = 0, previous_status = 0;
+			sensor_channel_get(dev, SENSOR_CHAN_AML,
+							aml_data);
+
+			if (aml_data[0].val1 != 0)
+				printf("[%s]: AML Gestures: %s \n",
+					   now_str(),			   
```

Comment:
Gestures are not exclusive.
Should be 
if(aml_data[0].val1 && ICM42670S_AML_SWIPE_LEFT) printf("Swipe left");
if(aml_data[0].val1 && ICM42670S_AML_SWIPE_RIGHT) printf("Swipe right");
...